src/components/Card/Card.tsx

```diff
@@ -6,20 +6,13 @@ import styles from "./Card.module.css";
 interface CardProps {
   id: string;
   name: string;
-  currentCohort: number;
   cohorts: number[];
   grade: Grade;
 }
 
-export default function Card({
-  id,
-  name,
-  currentCohort,
-  cohorts,
-  grade,
-}: CardProps) {
+export default function Card({ id, name, cohorts, grade }: CardProps) {
   const cohortString =
-    cohorts && cohorts.length > 0 ? cohorts.join(", ") : currentCohort;
+    cohorts && cohorts.length > 0 ? cohorts.join(", ") : cohorts.at(-1);
```

Comment:
```suggestion
    cohorts.length > 0 ? cohorts.join(", ") : "";
```
cohorts가 undefined는 될 수 없으니 조건문은 제거해도 될거같은데 어떻게 생각하시나요?
그리고 suggestion에 임시로 cohorts.length가 0일 때 ""로 작성하긴했는데 length가 없는 경우가 존재할까요? 🧐